app/models/ingest_job.rb

```diff
@@ -716,7 +732,8 @@ def get_job_analytics
       trigger: trigger,
       jobStatus: failed? ? 'failed' : 'success',
       machineType: vm_info['machineType'],
-      bootDiskSizeGb: vm_info['bootDiskSizeGb']
+      bootDiskSizeGb: vm_info['bootDiskSizeGb'],
+      exitStatus: exit_code #takes integer value from exit_code method
```

Comment:
This example would clarify for maintainers how `exitStatus` is useful beyond `jobStatus`:

```suggestion
      exitStatus: exit_code # integer exit code from PAPI, e.g. `137` for out of memory (OOM)
```